src/db-fixture-pg.ts

```diff
@@ -31,108 +21,100 @@ export class PGFixture
         let tool1_callback = (tool_id, tool_name, area_id, member_id) => {
             this.tool1ID = tool_id;
             this.tool1Name = tool_name;
-            this.addTool( db, area_id, member_id,
-                (tool2_id, tool2_name) => tool2_callback(
-                    tool2_id, tool2_name, area_id, member_id )
+            this.addTool(db, area_id, member_id, (tool2_id, tool2_name) =>
+                tool2_callback(tool2_id, tool2_name, area_id, member_id),
             );
         };
         let member_callback = (member_id, area_id) => {
-            this.addTool( db, area_id, member_id,
-                (tool1_id, tool1_name) => tool1_callback(
-                    tool1_id, tool1_name, area_id, member_id )
+            this.addTool(db, area_id, member_id, (tool1_id, tool1_name) =>
+                tool1_callback(tool1_id, tool1_name, area_id, member_id),
             );
         };
-        let area_callback = (area_id) => {
-            this.addMember( db,
-                (member_id) => member_callback( member_id, area_id ) );
+        let area_callback = area_id => {
+            this.addMember(db, member_id =>
+                member_callback(member_id, area_id),
+            );
         };
 
-        this.addArea( db, area_callback );
+        this.addArea(db, area_callback);
```

Comment:
It shouldn't collapse whitespace like this.  Whitespace is _good_.

Reply:
I didn't see an option for this in prettier.